install/installation-kubernetes.md

```diff
@@ -0,0 +1,175 @@
+
+<highlight type="note">
+If you configured the user credentials in the `my_values.yaml` file, you don't
+need to decode the passwords. In the following section replace `<MY_NAME>` with
+the name that you provided during the installation.
+</highlight>
+
+1. Get the name of the host to connect to:
+    ```bash
+    kubectl get service/<MY_NAME>
+    ```
+1. Decode the `admin` user password `PGPASSWORD_ADMIN` that was generated during
+   the Helm installation:
+    ```bash
+    PGPASSWORD_ADMIN=$(kubectl get secret --namespace default 
+    <MY_NAME>-credentials -o jsonpath="{.data.PATRONI_admin_PASSWORD}" | base64 --decode)
+    ``` 
+1. **OPTIONAL** Decode the super user password `PGPOSTGRESPASSWORD` that was
+   generated during the Helm installation:
+    ```bash
+    PGPASSWORD_POSTGRES=$(kubectl get secret --namespace default 
+    <MY_NAME>-credentials -o jsonpath="{.data.PATRONI_SUPERUSER_PASSWORD}" | base64 --decode)
+    ```
+1. Connect with psql as `admin` user:
+    ```bash
+    kubectl run -i --tty --rm psql --image=postgres \
+      --env "PGPASSWORD=$PGPASSWORD_ADMIN" \
+      --command -- psql -U admin \
+      -h <MY_NAME>.default.svc.cluster.local postgres
+    ```
+    
+</procedure>
+
+<procedure>
+
+### Connecting to TimescaleDB from inside the cluster
+1. Get the Pod on which TimescaleDB is installed:
+   ```bash
+    MASTERPOD="$(kubectl get pod -o name --namespace default -l release=test,role=master)"
+    ```
+1. Run `psql` inside the Pod containing the primary:
+    ```bash
+    kubectl exec -i --tty --namespace default ${MASTERPOD} -- psql -U postgres
+    ```
+
+</procedure>
+
+<procedure>
+
+## Create a database
+ After installing and connecting to TimescaleDB you can create a database,
+ connect to the database, and also verify that the TimescaleDB extension is
+ installed.
+
+### Creating a database
+1.  At the prompt, create an empty database. For example, to create a database
+    called `tsdb`:
+    ```sql
+    CREATE database tsdb;
+    ```
+1.  Connect to the database you created:
+    ```sql
+    \c tsdb
+    ```
+1. Verify that the TimescaleDB extension is installed by using the `\dx`
+command at the command prompt. It looks like this:
+    ```sql
+    tsdb=# \dx
+
+                                      List of installed extensions
+    Name     | Version |   Schema   |                            Description                            
+    -------------+---------+------------+-------------------------------------------------------------------
+    plpgsql     | 1.0     | pg_catalog | PL/pgSQL procedural language
+    timescaledb | 2.1.1   | public     | Enables scalable inserts and complex queries for time-series data
+    (2 rows)
+
+    (END)
+    ```
+
+</procedure>
+
+## Clean up
+You can use Helm to uninstall TimescaleDB on the Kubernetes cluster and clean up
+the Pods, Persistent Volume Claims (PVC), S3 backups, and more.
+
+### Cleaning up
+To remove the spawned Pods:
```

Comment:
```suggestion
To remove the spawned pods:
```

Reply:
@Loquacity In Kubernetes, the P in the Pods are always capitalized. [Pods](https://kubernetes.io/docs/concepts/workloads/pods/)

Reply:
Right, OK 👍